### PR TITLE
Merge Request to Rabbit C sharp

### PR DESCRIPTION
**Pull request by winhtaikaung**
Porting Rabbit-Converter Java to C sharp and build as Class Library Project

### CODE REVIEW

**Comment by winhtaikaung** [comment]
Add Assembly Information and Read Me

**Comment by yelinaung** [comment]
Good job! Ko @winhtaikaung.
A few things from me 
- Can you add `.gitignore` file. e.g something like [VisualStudio.gitignore](https://github.com/github/gitignore/blob/master/VisualStudio.gitignore) ?
- Can you add some Unit tests ? Please see [simple.json](https://github.com/Rabbit-Converter/Rabbit-Java/blob/master/src%2Ftest%2Fjava%2Fsamples.json) and how it's been [tested](https://github.com/Rabbit-Converter/Rabbit-Java/blob/master/src%2Ftest%2Fjava%2Forg%2Frabbitconverter%2Frabbit%2FRabbitTest.java) in Rabbit-Java

File: README.md [hunk 1]
```diff
@@ -1,4 +1,25 @@
 # Rabbit-CSharp
+![Logo](https://avatars3.githubusercontent.com/u/11961573?v=3&s=100)
+	
+### Zawgyi to Unicode <=> Unicode to Zawgyi converter for C#
```
Comment (yelinaung) [inline README.md]: This will start with `##` ( `h2` )

File: README.md [hunk 2]
```diff
@@ -1,4 +1,25 @@
 # Rabbit-CSharp
+![Logo](https://avatars3.githubusercontent.com/u/11961573?v=3&s=100)
+	
+### Zawgyi to Unicode <=> Unicode to Zawgyi converter for C# 
+	Requires :.Net framework 3.5
+	
+# Installation
```
Comment (yelinaung) [inline README.md]: This one too. `##` ( `h2` )

File: README.md [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+	Download Project as zip and Open Project in Visual Studio and Build 
+	
+# Usage
```
Comment (yelinaung) [inline README.md]: This one too. `##` ( `h2` )

File: README.md [hunk 4]
```diff
@@ -1,4 +1,25 @@
...
+	CSharp
+	Rabbit.uni2zg("သီဟိုဠ်မှ ဉာဏ်ကြီးရှင်သည် အာယုဝဍ်ဎနဆေးညွှန်းစာကို ဇလွန်ဈေးဘေးဗာဒံပင်ထက် အဓိဋ္ဌာန်လျက် ဂဃနဏဖတ်ခဲ့သည်။");
+    Rabbit.zg2uni("သီဟိုဠ္မွ ဉာဏ္ႀကီးရွင္သည္ အာယုဝဍ္ဎနေဆးၫႊန္းစာကို ဇလြန္ေဈးေဘးဗာဒံပင္ထက္ အဓိ႒ာန္လ်က္ ဂဃနဏဖတ္ခဲ့သည္။");
+
+    
+
+# Contributing
```
Comment (yelinaung) [inline README.md]: This one too. `##` ( `h2` )

File: RabbitConverter/Rabbit.cs [hunk 5]
```diff
@@ -0,0 +1,92 @@
+﻿/*The MIT License (MIT)
+
+Copyright (c) 2015 Htain Lin Shwe/Rabbit-Converter
```
Comment (yelinaung) [inline RabbitConverter/Rabbit.cs:11]: License shall be under `Rabbit-Converter` org

**Comment by yelinaung** [comment]
Pinging @winhtaikaung,

**Pull request closed without merging**